ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/TransactionInfoResult.java

```diff
@@ -22,17 +24,24 @@
 
   private final String hash;
   private final boolean isReceivedFromLocalSource;
+  private final Instant addedToPoolAt;
 
   public TransactionInfoResult(final TransactionInfo transactionInfo) {
     hash = transactionInfo.getHash().toString();
     isReceivedFromLocalSource = transactionInfo.isReceivedFromLocalSource();
+    addedToPoolAt = transactionInfo.getAddedToPoolAt();
   }
 
   @JsonGetter(value = "hash")
   public String getHash() {
     return hash;
   }
 
+  @JsonGetter(value = "addedToPoolAt")
+  public long getAddedToPoolAt() {
+    return addedToPoolAt.toEpochMilli();
```

Comment:
As discussed this should be ISO8601 format rather than epoch millis.

Reply:
cool -- have used toString on the instant which returns an ISO8601 string.